finality-aleph/src/substrate_network.rs

```diff
@@ -46,10 +47,10 @@ impl<B: Block, H: ExHashT> RequestBlocks<B> for Arc<NetworkService<B, H>> {
 const AUTHENTICATION_PROTOCOL_NAME: &str = "/aleph/1";
 
 /// Returns the canonical name of the protocol.
-pub fn protocol_name(protocol: &Protocol) -> Cow<'static, str> {
+pub fn protocol_name(protocol: &Protocol) -> ProtocolName {
     use Protocol::*;
     match protocol {
-        Authentication => Cow::Borrowed(AUTHENTICATION_PROTOCOL_NAME),
+        Authentication => ProtocolName::Static(AUTHENTICATION_PROTOCOL_NAME),
```

Comment:
nit: Can also be `AUTHENTICATION_PROTOCOL_NAME.into()`.

Reply:
done